groot/Source.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+
+int main() {
+
+	int user_input = 0;
+	std::cout << "Please enter a number: ";
+	std::cin >> user_input;
+
+	if (user_input < 0) {
+		std::cout << "Number cannot be negative";
+		return 1;
+	}
+
+	double sqrt = std::sqrt(user_input);
```

Comment:
'sqrt' is symbol in cmath.h
Overriding symbols is bad, find another name for this variable.